### PR TITLE
1.03 grammar style about

### PR DESCRIPTION
**Pull request by andyjohnson**
Nothing too tough in here.  A couple words were changed for intent and to be consistent with the rest of the document.

### CODE REVIEW

File: xAPI-About.md [hunk 1]
```diff
@@ -493,29 +495,33 @@ that an LRS can take. It's also possible that some LRSs will have no mechanism f
...
 It is recommended that Communities of Practice agree on how to use extensions for their particular use cases and utilize 
 Profiles whenever possible. Implementation details are covered in [4.1 Extensions](#miscext).
 
-The About Resource is another location where xAPI supports Extensions.  The LRS may find it useful to communicate features or 
+The About Resource is another location where xAPI supports extensions.  The LRS may find it useful to communicate features or 
 behaviors beyond this specification. The LRS can use extensions to the About Resource to facilitate this communication.
 
-Finally, the set of Resources implemented is not expected to be constrained by this document. Resources beyond 
-those listed in this specification can be implemented and co-exist with the Resources defined in this specification.
+Finally, the set of resources (endpoints) implemented is not expected to be constrained by this document. Resources beyond
```
Comment (garemoko) [inline xAPI-About.md]: See comment in the previous PR. Let's pick a name then stick to our guns.

File: xAPI-About.md [hunk 2]
```diff
@@ -493,29 +495,33 @@ that an LRS can take. It's also possible that some LRSs will have no mechanism f
...
+It is intended that [Communities of Practice (CoPs)](#def-community-of-practice) will define Verbs, Activity types, 
+contextual relationships, extensions, etc. 
+to be used in their profiles by providing unique identifiers wherever applicable. The CoP will define these identifiers and 
+their metadata in a [Vocabulary](#def-vocabulary). A Profile is a set of rules and vocabularies to implemented in addition to 
+xAPI for the particular use case being addressed. It is very important that such communities exist and share best practices. 
+For more information on publishing vocabularies, see the [Vocabulary Companion Specification](https://adl.gitbooks.io/companion-specification-for-xapi-vocabularies/content) and [Vocabulary Primer](https://adl.gitbooks.io/experience-xapi-vocabulary-primer/content/) documents.
 
-It is recommended that a Profile use a unique "category" within a Statement's context to refer to any Statement  
+It is recommended that a Profile use a unique "category" within a Statement's "context" property to refer to any Statement
```
Comment (garemoko) [inline xAPI-About.md]: I prefer the original. By adding "property" it makes this part sound more technical than it is, perhaps giving the idea that category is a direct property of context.

File: xAPI-About.md [hunk 3]
```diff
@@ -540,23 +546,20 @@ move those changes into the current specification and revision to 0.9.
...
+- Agent Objects now have exactly 1 uniquely identifying property, instead of at least one.
 
 ###### 0.95 to 1.0.0 (April 26, 2013) 
 Various refinements and clarifications including:
-- Adding attachments
+- Adding Attachments
 - Activity metadata is now stored as JSON rather than XML
 - Changes to voiding Statements
-- Clarification and naming of the Document APIs
+- Clarification and naming of the Document APIs (now Document Resources)
```
Comment (garemoko) [inline xAPI-About.md]: Can we change "now" to "later renamed to" to avoid suggesting that they changed to Document Resources  in 1.0.0?

**Comment by andyjohnson** [comment]
All good suggestions, all incorporated.  The "context" property is especially good because this section is intended to be less technical.

**Comment by garemoko** [comment]
+1

**Comment by ljwolford** [comment]
I did a review of the links that I sent to Andy to fix. Didn't notice any other glaring issues so once those are fixed +1

**Pull request merged**